Samples/Hast.Samples.Kpz.Algorithms/Readme.md

```diff
@@ -1,3 +1,49 @@
 # Hastlayer KPZ Algorithms
 
-Contains the actual algorithms of `Hast.Samples.Kpz`, the GUI app. This is necessary because due to Microsoft Catapult compatibility the GUI app needs to target x64. That however, will break tests relying on these algorithms.
+This is an FPGA implementation of the 3D [Kardar-Parisi-Zhang](https://en.wikipedia.org/wiki/Kardar%E2%80%93Parisi%E2%80%93Zhang_equation) surface growth simulation algorithm. Made in collaboration with the [Wigner Research Centre for Physics](http://wigner.mta.hu/en/).
+
+**Note:** This project is kept here because it's a useful example and also used for testing Hastlayer itself. Before the `` commit it also had a corresponding GUI project, but since that was Windows-only WinForms, we had to remove it.
```

Comment:
I think there is a commit hash missing from between the backticks?

Reply:
Yep!